src/js/view/view.js

```diff
@@ -417,12 +417,15 @@ define([
 
         function itemReady(item) {
             const provider = _model.getVideo();
-            if (provider && provider.getName().name.indexOf('flash') === 0) {
+            // Youtube, chromecast and flash providers do no support video tags
+            if (provider && /^(youtube|chromecast|flash)/.test(provider.getName().name)) {
                 return;
             }
-            const title = item.title || '';
             var videotag = _videoLayer.querySelector('video, audio');
-            videotag.setAttribute('title', title);
+            const dummyDiv = document.createElement('DIV');
```

Comment:
It sounds stingy, but do we really want to create a new element to decode titles for every item that plays? This element could be a static element used by all players for all string decoding.